resources/busola-migrator/values.yaml

```diff
@@ -8,14 +8,15 @@ global:
 deployment:
   image:
     repository: "eu.gcr.io/kyma-project/busola-migrator"
-    tag: "a5b820ae"
+    tag: "PR-11619"
     pullPolicy: IfNotPresent
   env:
     busolaURL: "https://dashboard.kyma.cloud.sap"
     oidcIssuerURL: "https://kyma.accounts.ondemand.com"
     oidcClientId: "12b13a26-d993-4d0c-aa08-5f5852bbdff6"
     oidcScope: "openid email"
     oidcUsePKCE: "true"
```

Comment:
If we no longer use oidc related values in the business logic I'd remove them from configs.